### PR TITLE
Move ViewComponent to separate GitHub organization

### LINKED ISSUE #1424: Move ViewComponent to its own org
I’d like to propose that we move ViewComponent to its own organization, `viewcomponent` for the following reasons:

1. So that ViewComponent’s identity can grow beyond the scope of just being a GitHub project. ViewComponent has > 165 contributors from across the industry, not just GitHub employees. 
2. So that we can formally extend ownership of the project to non-GitHub members of the community. This is currently not allowed under our organization policies.
3. Allow for other repos to be added to the project. For example, I’d be interested in moving VC::Form, Lookbook, demo repo and others into this proposed org.

What do y’all think?

### PR DESCRIPTION
**Pull request by joelhawksley**
Closes #1424

### What are you trying to accomplish?

This PR updates our docs ahead of moving ViewComponent to its own organization on GitHub ❤️

### CODE REVIEW

File: docs/CONTRIBUTING.md [hunk 1]
```diff
@@ -10,11 +10,11 @@ _ViewComponent is intended to be a safe, welcoming space for collaboration. By p
 
 Hi there! We're thrilled that you'd like to contribute to ViewComponent. Your help is essential for keeping it great.
 
-If you have any substantial changes that you would like to make, please [open an issue](http://github.com/github/view_component/issues/new) first to discuss them with us.
+If you have any substantial changes that you would like to make, please [open an issue](http://github.com/viewcomponent/view_componentmponent/issues/new) first to discuss them with us.
```
Comment (Spone) [inline docs/CONTRIBUTING.md:13]:
```suggestion
If you have any substantial changes that you would like to make, please [open an issue](http://github.com/viewcomponent/view_component/issues/new) first to discuss them with us.
```

**Review by Spone — CHANGES_REQUESTED** [review]

**Review by boardfish — APPROVED** [review]

**Review by Spone — APPROVED** [review]

File: docs/CHANGELOG.md [hunk 2]
```diff
@@ -10,6 +10,8 @@ nav_order: 5
 
 ## main
 
+**ViewComponent has moved to a new organization: [https://github.com/viewcomponent/view_component](https://github.com/viewcomponent/view_component). See [https://github.com/viewcomponent/view_componentmponent/issues/1424](https://github.com/viewcomponent/view_component/issues/1424) for more details.**
```
Comment (rmacklin) [inline docs/CHANGELOG.md:13]:
```suggestion
**ViewComponent has moved to a new organization: [https://github.com/viewcomponent/view_component](https://github.com/viewcomponent/view_component). See [https://github.com/viewcomponent/view_component/issues/1424](https://github.com/viewcomponent/view_component/issues/1424) for more details.**
```

File: docs/CONTRIBUTING.md (hunk 1 above)
Comment (rmacklin) [inline docs/CONTRIBUTING.md:13]:
```suggestion
If you have any substantial changes that you would like to make, please [open an issue](http://github.com/viewcomponent/view_component/issues/new) first to discuss them with us.
```

**Review by rmacklin — COMMENTED** [review]
Looks like 11d5ac8d6e13acf54eb1c2ab6e9359fbc254109c may have been unintentionally dropped, and the same typo was introduced in another file in b8efb570c24ee973295e798177421e2ff8116f57

**Pull request merged**